src/documentation/ToolkitDocumentation/api.ts

```diff
@@ -62,3 +100,18 @@ export const fetchToolkit = async (languageId: string): Promise<Toolkit> => {
   }
   return fallback;
 };
+
+export const getTopicAndActiveItem = (
+  toolkit: Toolkit,
+  topicId: string | undefined
+): [ToolkitTopic | undefined, ToolkitTopicEntry | undefined] => {
+  let activeItem: ToolkitTopicEntry | undefined;
+  const topic: ToolkitTopic | undefined = toolkit.contents?.find((t) => {
+    if (t.slug.current === topicId) {
+      return t;
+    }
+    activeItem = t.contents?.find((e) => e.slug.current === topicId);
+    return activeItem?.parent;
+  });
+  return [topic, activeItem];
+};
```

Comment:
Is there a better way to handle TypeScript here?

Reply:
We made it... different.